clang/test/SemaCXX/default-op-notusing-namespace.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %clang_cc1 -std=c++20 -fsyntax-only -verify %s
```

Comment:
These two should just be in the same test file, there is no reason to have two separate tests for this. Perhaps find an existing test file for this to live in?